BrWizardNavigation.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="row justify-center width-100 q-mt-lg">
+    <back-button
+      v-if="currentStepIndex > 0"
+      class="q-mr-md"
+      :disabled="blockBack"
+      @back="onBack()" />
+    <next-button
+      v-if="currentStepIndex < totalSteps - 1"
+      :class="{'q-ml-md': currentStepIndex > 0}"
+      :disabled="blockNext"
+      @next="onNext()" />
+    <finish-button
+      v-if="currentStepIndex === totalSteps - 1"
+      class="q-ml-md"
+      :disabled="blockFinish"
+      @finish="onFinish()" />
+  </div>
+</template>
+
+<script>
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+import NextButton from './NextButton.vue';
+import BackButton from './BackButton.vue';
+import FinishButton from './FinishButton.vue';
+
+export default {
+  name: 'BrWizardNavigation',
+  components: {NextButton, BackButton, FinishButton},
+  props: {
+    blockBack: {
+      type: Boolean,
+      required: true
+    },
+    blockFinish: {
+      type: Boolean,
+      required: true
+    },
+    blockNext: {
+      type: Boolean,
+      required: true
+    },
+    currentStepIndex: {
+      type: Number,
+      required: true
+    },
+    totalSteps: {
+      type: Number,
+      required: true
+    },
+  },
+  methods: {
+    async onNext() {
+      if(!await this._emitCancelableEvent('next', {
+        currentStepIndex: this.currentStepIndex
+      })) {
+        return;
```

Comment:
idk what's going on with this pattern.  Are these return statements necessary?  If so, can you please explain why so I'll know for future.

Reply:
I was confused looking at this initially too, until @dlongley explained this was some common web pattern. Don't remember too much of the conversation, so @dlongley might have to help clarify the cancelable event pattern.

Reply:
I note that the single `l` in cancelable is britspeak: https://www.dictionary.com/browse/cancelled

Reply:
@mattcollier, I think you have that backwards ... Americans prefer one `l` and the British prefer two.